jekyll/_cci2_ja/migrating-from-aws.adoc

```diff
@@ -266,4 +266,4 @@ jobs:
 . https://circleci.com/docs/ja/artifacts/[アーティファクト]
 . https://circleci.com/docs/ja/caching/[キャッシュ]
 . https://circleci.com/docs/ja/triggers/#section=jobs[トリガー]
-. https://circleci.com/docs/ja/optimizations/#section=projects[パフォーマンスオプション]
+. https://circleci.com/docs/ja/optimizations/#section=projects[パフォーマンスオプション]
```

Comment:
The page title for the link was 「最適の概要」wondering if this might be better choice to match with the title. Or found a similar translation as 「キャッシュ、ワークスペース、並列実行などの最適化」on the following diff.  https://github.com/circleci/circleci-docs/pull/7257/files#diff-6b408f455151a5c3cc9f498567cf04a06af94a7971544cfe708000f95209ff9eR308